microsoft-edge/test-and-automation/webhint.md

```diff
@@ -0,0 +1,19 @@
+---
+description: About the webhint extension for Visual Studio Code.
+title: webhint extension for Visual Studio Code
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/03/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, vs code, visual studio code, webhint
+---
+# webhint extension for Visual Studio Code
+
+Use [webhint](https://webhint.io), a customizable linting tool, to improve the accessibility, performance, cross-browser compatibility, PWA compatibility, and security of your site.  The webhint extension checks your code for best practices and common errors. This open-source project, initially developed by the Microsoft Edge team, is now part of the [OpenJS Foundation](https://openjsf.org).  The Microsoft Edge team continues to contribute to webhint alongside web developers in the community.
+
+Identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more by adding the webhint extension for Visual Studio Code.  Hints appear as inline underlines and are summarized in the **Problems** pane.
+
+:::image type="content" source="../visual-studio-code/media/webhint-extension.png" alt-text="The webhint extension for Visual Studio Code.":::
```

Comment:
can we add redlines around what is helpful to focus on? Otherwise should we remove it?